### PR TITLE
Remove usage of parity_bytes

### LINKED ISSUE #57: Remove dependency on parity_bytes

### PR DESCRIPTION
**Pull request by aarlt**
Closes #57

### CODE REVIEW

File: bls12pairing/Cargo.toml [hunk 1]
```diff
@@ -10,8 +10,7 @@ edition = "2018"
 
 [dependencies]
 ewasm_api = "0.9"
-ethereum-bls12 = { git = "https://github.com/ewasm/ethereum-bls12.rs", tag = "0.1.1" }
-parity-bytes = { git = "https://github.com/paritytech/parity-common", version = "0.1.0" }
+ethereum-bls12 = { git = "https://github.com/aarlt/ethereum-bls12.rs", branch = "remove_parity_bytes" }
```
Comment (axic) [inline bls12pairing/Cargo.toml]: This shouldn't use a branch, but a version when merging.

Comment (axic) [reply bls12pairing/Cargo.toml]: This should use version 0.1.2.

File: ecadd/Cargo.toml [hunk 2]
```diff
@@ -10,8 +10,7 @@ edition = "2018"
 
 [dependencies]
 ewasm_api = "0.9"
-ethereum-bn128 = { git = "https://github.com/ewasm/ethereum-bn128.rs", tag = "0.0.1" }
-parity-bytes = { git = "https://github.com/paritytech/parity-common", version = "0.1.0" }
+ethereum-bn128 = { git = "https://github.com/aarlt/ethereum-bn128.rs", branch = "remove_parity_bytes" }
```
Comment (axic) [inline ecadd/Cargo.toml]: Should use 0.1.0 for these.

Comment (aarlt) [reply ecadd/Cargo.toml]: done

**Comment by axic** [comment]
Thanks for the PRs, this is finally done and merged :wink:

**Pull request merged**